extensions/vscode/README.md

```diff
@@ -1,11 +1,60 @@
-# Posit Publisher
+# Posit Publisher - Deploy to Posit Connect
 
-[Posit Publisher](https://github.com/posit-dev/publisher) is an extension for
-[Visual Studio Code](https://code.visualstudio.com/) that provides an easy
-way to deploy to [Posit Connect](https://posit.co/products/enterprise/connect/).
+[Posit Publisher](https://github.com/posit-dev/publisher) is a code-first tool
+for deploying to
+[Posit Connect](https://posit.co/products/enterprise/connect/).
```

Comment:
Suggest including the types of things supported?

Posit Publisher is a code-first tool with a push-button user interface for deploying the things you build in R and Python to Posit Connect.

Supported content types (frameworks) (From the changelog?)
- Python content: APIs, applications, and notebooks
- Quarto content: .qmd and .Rmd files. Support for embedded Shiny apps
- R content: Shiny apps, RMarkdown, Plumber APIs, and Rmd with embedded Shiny apps

Reply:
I was a bit worried about listing everything since it would be quite long - see the types we support from the [Configuration doc](https://github.com/posit-dev/publisher/blob/main/docs/configuration.md#type).

But it did feel strange to leave it out. Thanks for the suggestion, I'll get something added!

Reply:
I included this verbatim. I tried to come up with a more succinct opener but its nice to include "code-first" and "push-button" into the first line. Let me know if you agree that it is a little cumbersome, and we can try to reduce it.